### PR TITLE
adding OVHcloud

### PR DESCRIPTION
**Pull request by mhurtrel**
Adding MKS OVHcloud offering

### CODE REVIEW

File: content/fr/docs/setup/pick-right-solution.md [hunk 1]
```diff
@@ -95,6 +95,8 @@ workloads, et plus encore.
 
 * [Oracle Container Engine for Kubernetes](https://docs.us-phoenix-1.oraclecloud.com/Content/ContEng/Concepts/contengoverview.htm) est un service entièrement géré, évolutif et hautement disponible que vous pouvez utiliser pour déployer vos applications conteneurisées dans le cloud.
 
+* [OVHcloud Managed Kubernetes Service](https://www.ovhcloud.com/fr/public-cloud/kubernetes/) est un service gratuit, entièrement géré, évolutif et hautement disponible pour héberger vos applications conteneurisées sur le Cloud Public OVHcloud. Il est disponible sur plusieurs continents.
```
Comment (mhamdisemah) [inline content/fr/docs/setup/pick-right-solution.md]: it's not completely free

Comment (mhurtrel) [reply content/fr/docs/setup/pick-right-solution.md]: Like Google GKE or Azure AKS, OVHcloud offers the Managed Kubernetes Service tottaly for free (no cost for the kubernetes service, all the components are hosted for free, you dont pay any traffic or api calls). Of course, you pay the ressources that are orchestrated though kubernetes ( worker-only nodes, peristent volumes, external load balancer) at standard price. 

I think it is worth noting noting it is free, compared to paid alternatives in the same market such as AWS EKS.

Comment (mhamdisemah) [reply content/fr/docs/setup/pick-right-solution.md]: you can have that Google Kubernetes Engine it not indicated Free , and all others solutions there is no one indicated Free

Comment (mhurtrel) [reply content/fr/docs/setup/pick-right-solution.md]: Indeed, thats a good point. I removed this information from my PR

**Comment by mhamdisemah** [comment]
Hi @yastij , can review the change to approve ?

**Comment by sftim** [comment]
The upstream page is gone. It does not seem appropriate to make changes that aren't drawn from the original (English) localization.

/hold

**Pull request closed without merging**